docs/platforms/javascript/guides/angular/features/component-tracking.mdx

```diff
@@ -44,76 +33,39 @@ export class AppModule {}
+<articles-list trace="articles-list"></articles-list>
+<app-footer trace="footer"></app-footer>
 ```
 
-<Alert level="info" title="Compatibility">
+## Using `TraceClass`
 
-If you're using version 6 of the Angular SDK, using `TraceDirective` or `TraceModule` causes a
-compiler error at application compile time of your Angular application. This is a [known issue](https://github.com/getsentry/sentry-javascript/issues/3282)
-of our Angular SDK v6 and it was [fixed](https://github.com/getsentry/sentry-javascript/issues/4644)
-in version 7. We recommend upgrading to the latest Angular SDK version.
-Otherwise, please use the other options (`TraceClassDecorator` and `TraceMethodDecorator`)
-below to track your Angular components.
+The `TraceClass` decorator tracks the duration between the `OnInit` and `AfterViewInit` lifecycle hooks in components. It adds spans called **`ui.angular.init`** to the currently active transaction. In contrast to the [`trace` directive](#using-the-trace-directive), `TraceClassDecorator` tracks all instances of the component(s) you add it to, creating spans for each component instance.
 
-</Alert>
-
-## Using `TraceClassDecorator`
-
-`TraceClassDecorator` tracks the duration between the `OnInit` and `AfterViewInit` lifecycle hooks in components. It adds spans called **`ui.angular.init`** to the currently active transaction. In contrast to [`TraceDirective`](#using-tracedirective), `TraceClassDecorator` tracks all instances of the component(s) you add it to, creating spans for each component instance.
-
-Just add `TraceClassDecorator` to the components you want to track:
-
-```javascript {filename:header.component.ts} {tabTitle: Angular 12+}
-import { Component } from "@angular/core";
-import * as Sentry from "@sentry/angular-ivy";
-
-@Component({
-  selector: "app-header",
-  templateUrl: "./header.component.html",
-})
-@Sentry.TraceClassDecorator()
-export class HeaderComponent {
-  // ...
-}
-```
+Just add `TraceClass` decorator with a component name to the components you want to track:
 
-```javascript {filename:header.component.ts} {tabTitle: Angular 10/11}
+```typescript {filename:header.component.ts} {2,8}
 import { Component } from "@angular/core";
 import * as Sentry from "@sentry/angular";
 
 @Component({
   selector: "app-header",
   templateUrl: "./header.component.html",
 })
-@Sentry.TraceClassDecorator()
+@Sentry.TraceClass({ name: "Header" })
 export class HeaderComponent {
   // ...
 }
 ```
 
-## Using `TraceMethodDecorator`
-
-`TraceMethodDecorator` tracks specific component lifecycle hooks as point-in-time spans. The added spans are called **`ui.angular.[methodname]`** (like, `ui.angular.ngOnChanges`). For example, you can use this decorator to track how often component changes are detected during an ongoing transaction:
+Note: Due to code minification in production builds, you need to pass a `name` property to the `TraceClass` decorator. Otherwise, spans will refer to the component as `<unnamed>`
```

Comment:
```suggestion
Note: Due to code minification in production builds, you need to pass a `name` property to the `TraceClass` decorator. Otherwise, spans will refer to the component as `<unnamed>`.
```